lib/hammer_cli_csv/content_hosts.rb

```diff
@@ -167,12 +167,12 @@ def import
       end
 
       def import_remotely
-        @api.resource(:csv).call(:import_content_hosts, {
+        task_progress(@api.resource(:csv).call(:import_content_hosts, {
             'content' => ::File.new(::File.expand_path(option_csv_file), 'rb')
         }, {
             :content_type => 'multipart/form-data',
             :multipart => true
-        })
+        }))
```

Comment:
this would be a lot easier to read if you placed your arguments into variables.

```ruby
data = { :content => ::File.new(::File.expand_path(option_csv_file), 'rb') }
header = { :content_type => 'multipart/form-data',
           :multipart => true }

response = @api.resource(:csv).call(:import_content_hosts, data, header)
task_progress(response)
```

isn't that much easier to read?